src/components/ThreadEnvelope.vue

```diff
@@ -29,7 +29,7 @@
 				:email="envelope.from[0].email"
 				:display-name="envelope.from[0].label"
 				:disable-tooltip="true"
-				:size="40" />
+				:size="44" />
```

Comment:
Note that increasing the avatar size also needs more text indent of the mail text, so that sender name and mail body is aligned. :)

Reply:
![image](https://user-images.githubusercontent.com/3595562/201829346-550cd5d1-7fb6-468f-94e2-08ba0975056c.png)

Like this?